deploy.sh

```
@@ -0,0 +1,60 @@
+      rm "../../$PID_FILE"
+  fi
+
+  # ===== 새 앱 실행 =====
+  echo ">>> 새 앱 실행: $JAR_NAME"
+  nohup java -jar $JAR_NAME > "../../$LOG_FILE" 2>&1 &
+
+  # ===== PID 저장 =====
+  echo $! > "../../$PID_FILE"
+
+  # ===== 배포 성공 여부 분기 =====
+  if [ $? -eq 0 ]; then
```

Shellcheck (0.10.0)

[warning] This $? refers to echo/printf, not a previous command. Assign to variable to avoid it being overwritten.

(SC2320)